connect_ext_test.go

```diff
@@ -1393,6 +1467,305 @@ func TestBidiStreamServerSendsFirstMessage(t *testing.T) {
 	})
 }
 
+func TestStreamForServer(t *testing.T) {
+	t.Parallel()
+	newPingServer := func(pingServer pingv1connect.PingServiceHandler) (pingv1connect.PingServiceClient, *httptest.Server) {
+		mux := http.NewServeMux()
+		mux.Handle(pingv1connect.NewPingServiceHandler(pingServer))
+		server := httptest.NewUnstartedServer(mux)
+		server.EnableHTTP2 = true
+		server.StartTLS()
+		t.Cleanup(server.Close)
+		client := pingv1connect.NewPingServiceClient(
+			server.Client(),
+			server.URL,
+		)
+		return client, server
+	}
+	t.Run("not-proto-message", func(t *testing.T) {
+		client, server := newPingServer(&pluggablePingServer{
+			cumSum: func(ctx context.Context, stream *connect.BidiStream[pingv1.CumSumRequest, pingv1.CumSumResponse]) error {
+				return stream.Conn().Send("foobar")
+			},
+		})
+		t.Cleanup(server.Close)
+		stream := client.CumSum(context.Background())
+		assert.Nil(t, stream.Send(nil))
+		_, err := stream.Receive()
+		assert.NotNil(t, err)
+		assert.Equal(t, connect.CodeOf(err), connect.CodeInternal)
+		assert.Nil(t, stream.CloseRequest())
+		t.Parallel()
```

Comment:
Here and in the subsequent subtests, I think the `t.Parallel()` should probably be at the top.

Reply:
copypasta issue 🙃